docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -66,38 +65,38 @@ To build, first create a build directory ("build" in the example below) and chan
 Then run `cmake` to configure the build and generate the Makefiles, and then build by running `ninja`.
 
 To build debug enclaves:
-```cmd
-cd C:\openenclave
-mkdir build\x64-Debug
-cd build\x64-Debug
-cmake -G Ninja -DNUGET_PACKAGE_PATH=C:\oe_prereqs -DCMAKE_INSTALL_PREFIX:PATH=C:\openenclave ..\..
+```powershell
+cd C:/openenclave
+mkdir build/x64-Debug
+cd build/x64-Debug
+cmake -G Ninja -DNUGET_PACKAGE_PATH=C:/oe_prereqs -DCMAKE_INSTALL_PREFIX=install ../..
 ninja
 ```
 
-Later, using the `ninja install` command will install the SDK in C:\openenclave. To choose a different location, change the value specified for CMAKE_INSTALL_PATH.
+Later, using the `ninja install` command will install the SDK in `C:/openenclave/build/x64-Debug/install`. To choose a different location, change the value specified for `CMAKE_INSTALL_PATH`, but note that the samples tests will likely break.
```

Comment:
@andschwa Thanks for these edits. Is there a fundamental reason why we need to choose a relative path for the install and not an absolute path? Why do sample tests fail when we choose an absolute path? Is there a way to fix this?

Reply:
Yes, it's the only way that `test-samples.cmake` will work on Windows. If we fixed it to work with absolute paths, then it would break with relative paths, and running `ctest` would pollute a hard-coded absolute path, so I don't recommend that. I think we should keep it working the same way as it works on Linux.

Reply:
Hi @radhikaj following the guide as is - on Windows caused all samples to fail:  installdir when not used with relative path would ultimately cause installdir to be incorrect:

![image](https://user-images.githubusercontent.com/56414747/69167131-6f921800-0ac2-11ea-82fb-37f86b51f4bd.png)